platform/web/detect.py

```diff
@@ -256,7 +256,7 @@ def configure(env: "SConsEnvironment"):
         env.Append(CCFLAGS=["-sUSE_PTHREADS=1"])
         env.Append(LINKFLAGS=["-sUSE_PTHREADS=1"])
         env.Append(LINKFLAGS=["-sDEFAULT_PTHREAD_STACK_SIZE=%sKB" % env["default_pthread_stack_size"]])
-        env.Append(LINKFLAGS=["-sPTHREAD_POOL_SIZE=\"Module['emscriptenPoolSize']||8\""])
+        env.Append(LINKFLAGS=["-sPTHREAD_POOL_SIZE='Module['emscriptenPoolSize'] ?? navigator.hardwareConcurrency'"])
```

Comment:
This needs to be checked on Windows, see:
* https://github.com/godotengine/godot/pull/107112